docs/src/main/markdown/customizations.md

```diff
@@ -529,8 +529,8 @@ In addition to primitive values, you can customize enums and messages as well.
 
 For more examples, see:
 
-- [`custom_types.proto`](https://github.com/scalapb/ScalaPB/blob/master/e2e/src/main/protobuf/custom_types.proto)
-- [`PersonId.scala`](https://github.com/scalapb/ScalaPB/blob/master/e2e/src/main/scala/com/thesamet/pb/PersonId.scala)
+- [`custom_types.proto`](https://github.com/scalapb/ScalaPB/blob/master/e2e-withjava/src/main/protobuf/collection_types.proto)
```

Comment:
should the link be to custom_types and not collection_types?

Reply:
🤦🏻  sure, thanks